training/data_transformer.py

```diff
@@ -42,8 +45,24 @@ class MetaData(object):
     scale_other = None # vector<float>
     joint_others = None # vector<Joints>
 
-    def __init__():
+    def __init__(dataset, img_size, is_validation, num_other_people, people_index, annolist_index,write_number,total_write_number,epoch, objpos, scale_self, joint_self,objpos_other,scale_other, joint_others):
```

Comment:
Remove this change, we don't need this. We set all the metadata values by directly accessing their fields.